contract/tests/test_contract.py

```diff
@@ -20,6 +20,7 @@ class TestContractBase(common.SavepointCase):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```

Comment:
Please don't do that, as what this lighten the tests, may cause other side effects (see for example #618 that tests this kind of things)

Reply:
This is true only if you test messages, which is not the case here...

Reply:
I have put you an example on something that is coming to 13.0, so please don't do it, and it's totally out of scope of this PR. What is the gain in performance with this anyway (out of curiosity)?

Reply:
Ok, I'll remove it.

For instance, without that we are at 24.76s, 20259 queries.
With: 20.40s, 18229 queries

As said, this shouldn't be used if we test messages for chatter. Instead, it should be enabled.